vocabularies/PDF.xml

```diff
@@ -80,6 +80,12 @@
             <String>Is padding (left, right, bottom, top) supported?</String>
           </Annotation>
         </Property>
+        <Property Name="HeaderFooter" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Header and Footer of document" />
```

Comment:
```suggestion
          <Annotation Term="Core.Description" String="Whether the document contains Header and Footer" />
```
This boolean property only controls whether header and footer exist, not how they look, right?

Reply:
Hi Heiko,

this annotation tells the client if the backend is capable to add a header or a footer. A requirement is that the "PDF API" is able to do this and this is only supported with a certain Backend version.

To control the page number, data/time stuff in a header or footer the client has to do a Create request on 
<EntityType Name="DocumentDescription"> of our common service in OData.

Reply:
Will you then rephrase the description as I suggested?